functionalTest/test/specs/overview.js

```diff
@@ -7,6 +8,101 @@ describe("Covid-19 Dashboard", () => {
     expect(browser).toHaveTitle(title);
     expect(dashboard.imgLogo).toBeDisplayed();
     expect(dashboard.headingTitle).toHaveText("Scottish COVID-19 Statistics");
-    expect(dashboard.headlineBanner).toHaveTextContaining("Phase 3");
+    expect(dashboard.headlineBanner).toHaveTextContaining(
+      "Regional Restrictions to help fight the pandemic"
+    );
+  });
+});
+
+describe("geoHeatMap Area Count", () => {
+  it("Health Board Area geoHeatMap-->Cases", () => {
+    dashboard.open();
+    dashboard.selectHealthBoardsButton.click();
```

Comment:
One thing here - the cases button and the health boards button could both be broken and the tests would still pass. In the heatmap test spec file there are tests which first click away (and check that worked) diring the test setup, before clicking the buttons to be tested and checking expectations.

Reply:
I added some expect condition after each button click. Please review. Kindly let me know if anything needs to be added.

Reply:
Yeah, the problem still exists. If the region type is already at 'health boards' and the value type is already at 'cases' then clicking the buttons doesn't do anything, and the expects still don't prove that those buttons work. Take a look at how the heatmap test spec file does it.

Reply:
@donal-s , I will need your guidance to use the same method in this file.